stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MessageReplyView.kt

```diff
@@ -144,10 +147,13 @@ internal class MessageReplyView : FrameLayout {
                     }
                     setTint(color)
                 }
-                isMine -> {
+                message.isMine() -> {
                     paintStyle = Paint.Style.FILL_AND_STROKE
-                    val color =
+                    val color = if (isMine) {
```

Comment:
This condition is a bit weird - we check if hte message is mine and then proceed to check again for the BG color?

Reply:
this is connected to the colour change down below. Before there was a bug where the avatar side was not determined properly. Now i changed that the users reply has the same colour as his messages so when someone else replies to current users message the MessageReply is grey. If we set the colour as you commented back, we then need to show the border, otherwise there is no UI distinction for the quoted message. eg: 
![Screenshot 2022-05-26 at 11 21 57](https://user-images.githubusercontent.com/38032787/170458973-17317a67-849a-4aa9-98b6-c95dc4f97ef4.png)

And if we change the colour down below to match the regular message colour then invert of the message above applies, grey background for reply inside grey message.

The check is as follows: message.isMine for the replied to message and is mine for the message containing the reply. If bot are true set different background for replied to message.